### DISCUSSION TITLE
Augmented FACE AR

### DISCUSSION THREAD

**Discussion by Amitdygon** [google/filament#5990] [Q&A]
Hello Team 
Is their any way in Filament to put a 3D model on the face like Google Augmented Ar core. If its possiblen than help me out to achieve this .

Thanks

**Comment by ouyangpeng** [comment]
I have the same business requirement.

**Reply by Amitdygon** [reply]
I have achieved this using  https://github.com/SceneView/sceneform-android this. its also provide morph targets to animate the blink and smile.
